dockernet/dockerfiles/node_init

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
remove this file

Reply:
Removing this now as we have added commands to Dockerfile